package.json

```diff
@@ -3,14 +3,19 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node server.js",
-    "watch": "npm run watch:server & npm run watch:client",
-    "watch:server": "nodemon server.js",
-    "watch:client": "webpack -w --progress --colors",
-    "dev": "webpack-dev-server",
-    "build": "NODE_ENV=production webpack -p",
+    "start": "node build/server.js",
+    "build": "mkdir -p build && npm run build:static && npm run build:client && npm run build:server",
+    "build:client": "webpack --progress --config webpack.client.config.js",
```

Comment:
for the build scripts, i think we need to add `-p` (for production) to the webpack command so that it properly minifies / optimizes